src/components/grids/DataGrid/index.tsx

```diff
@@ -200,6 +204,8 @@ export default function DataGrid({
     }
   );
 
+  const finalSortBy = useMemo(() => sortBy.filter((column) => column.desc !== undefined), [sortBy]) as SortBy;
```

Comment:
suggestion - maybe "filteredSortBy" would be more clear? Or perhaps a comment on why we need to filter would be helpful! I'd have assumed that sortBy only included columns that had a defined .desc value.